pkg/client/preflight.go

```diff
@@ -54,20 +54,31 @@ const (
 	kubeSystemNamespace = "kube-system"
 	kubeDNSLabelKey     = "k8s-app"
 	kubeDNSLabelValue   = "kube-dns"
+	coreDNSLabelValue   = "coredns"
```

Comment:
What deployment tool is this sets this label?  Kubeadm overloads for the kube-dns label w/coreos.

Reply:
I think we should check kubespray and make a call, I'll try to see if they do the same.

Reply:
There is a fresh PR approved for adding coredns to kubespray. There I dont see any overwriting action

https://github.com/kubernetes-incubator/kubespray/pull/2462

Reply:
@pipo02mix - It appears they add `k8s-app : coredns` and that seems like a good addition